src/main/java/app/controllers/OrderController.java

```diff
@@ -94,7 +102,26 @@ public static int calculateTotalPrice(List<OrderItem> basket) {
         return totalPrice;
     }
 
-    private static void placeOrder() {
+    public static void placeOrder(Context ctx, ConnectionPool connectionPool) {
+        try {
+
+            User userId = ctx.sessionAttribute("currentUser");
+            List<OrderItem> basket = ctx.sessionAttribute("basket");
+
+            if (userId != null && basket != null && !basket.isEmpty()) {
+
+                Order newOrder = new Order(-1,userId.getUserId(), basket, "PENDING", LocalDateTime.now());
```

Comment:
Add a constructor to Order that does not take in orderId as a parameter to remove clutter in this method